include/openssl/e_os2.h

```diff
@@ -350,6 +350,16 @@ typedef unsigned __int64 uint64_t;
 #  endif
 # endif
 
+# ifdef __STDC_VERSION__
+#  if (__STDC_VERSION__ >= 201112L)
+#   include <stdnoreturn.h>
+#  endif
```

Comment:
There's a logic flaw here that will mean that for any `__STDC_VERSION__` lower than 201112L, there will be no `noreturn`